docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -376,7 +427,30 @@ If you're working with time window partitions, you can also use the `asset_parti
 
 #### Handling partition mappings <Experimental />
```

Comment:
The extra detail that you've added about partition mappings is helpful - this section was a little bit sparse.